package/MDAnalysis/coordinates/memory.py

```diff
@@ -196,20 +196,25 @@ class Timestep(base.Timestep):
     """
     Timestep for the :class:`MemoryReader`
 
-    Overrides the positions property in
-    :class:`MDAnalysis.coordinates.base.Timestep` to use avoid
-    duplication of the array.
-
+    Compared to the base :class:`base.Timestep`, this version has an additional
+    :meth:`replace_positions_array` method. This property replaces the array of
+    positions, while the :attr:`positions` property replaces the content of the
+    array.
     """
-
-    @property
-    def positions(self):
-        return base.Timestep.positions.fget(self)
-
-    @positions.setter
-    def positions(self, new):
+    def replace_positions_array(self, new):
```

Comment:
It sounds as if this should not really be used by the average user. Perhaps make it underscore-named? `_replace_positions_array()`?